ansible/roles/test/files/ptftests/fast-reboot.py

```diff
@@ -381,6 +382,7 @@ def __init__(self):
         self.check_param('min_bgp_gr_timeout', 15, required = False)
 
         # Default settings
+        self.ping_dut_pkts = 100
```

Comment:
I'd suggest to use lower value, like 10 or 20. We need higher values in the data plane tests because of requirements.